app/tests/integration/nimbus/conftest.py

```diff
@@ -38,3 +44,27 @@ def _verify_url(request, base_url):
         retries = Retry(backoff_factor=0.1, status_forcelist=[500, 502, 503, 504])
         session.mount(base_url, HTTPAdapter(max_retries=retries))
         session.get(base_url, verify=False)
+
+
+@pytest.fixture
+def default_data():
+    return BaseDataClass(
+        hypothesis="smart stuff here",
+        application="DESKTOP",
+        public_description="description stuff",
+        branches=[
+            BaseBranchDataClass(
+                name="name 1",
+                description="a nice experiment",
+                config="No Feature Firefox Desktop",
+            )
+        ],
+        metrics=BaseMetricsDataClass(primary_outcomes=[], secondary_outcomes=[]),
+        audience=BaseAudienceDataClass(
+            channel="Nightly",
+            min_version=80,
+            targeting="TARGETING_MAC_ONLY",
+            percentage=50.0,
+            expected_clients=50,
+        ),
+    )
```

Comment:
Fixed in https://github.com/mozilla/experimenter/pull/6099/commits/b71f79a2ed8b031ae43b0b75ca14f4359eab64b8

Reply:
One more thing I'd like to do here is using the value of the `channel` select instead of the name for consistency reasons. I'll wait for this until #6132 is resolved though.